nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/TreeCommentManager.java

```diff
@@ -136,10 +135,9 @@ public Comment getExternalComment(CoreSession session, String entityId) {
         DocumentModel commentModel = getExternalCommentModel(session, entityId);
         // Check the read permissions using the given session
         checkReadCommentPermissions(session, commentModel.getRef());
-        return Framework.doPrivileged(() -> COMMENT_DOC_TYPE.equals(commentModel.getType())
-               ? Comments.newComment(commentModel)
-               : Comments.newAnnotation(commentModel)
-        );
+        return Framework.doPrivileged(
+                () -> COMMENT_DOC_TYPE.equals(commentModel.getType()) ? Comments.newComment(commentModel)
+                        : Comments.newAnnotation(commentModel));
```

Comment:
you can add some ` //` at the end of each line this let the code more readable

Reply:
rebased on 28253